src/anomalib/metrics/per_image/_binclf_curve_numba.py

```diff
@@ -0,0 +1,113 @@
+"""Binary classification matrix curve (NUMBA implementation of low level functions).
```

Comment:
it`'s similar to the `_` attributes inside a module

a function `_func` in `module.py` is supposed to be used within  the `module` scope, right?

similarly, the `_validate` is private to `per_image/`, so it can be used across other modules inside the subpackage `per_image` but not out of it